docs/pages/database-access/guides/postgres-redshift.mdx

```diff
@@ -8,253 +8,230 @@ videoBanner: UFhT52d5bYg
-![Redshift cluster endpoint](../../../img/database-access/guides/redshift/cluster-endpoint@2x.png)
-</Figure>
-
-You'll only need the `host:port` portion of the endpoint.
-
-**Take note of database user name and database name you will use to connect.**
-
-If you haven't created any database users, default database admin user name and
-maintenance database are displayed on the `Database configurations` panel of the
-`Properties` tab:
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
 
-<Figure align="center" bordered caption="Redshift cluster user/database">
-![Redshift cluster user/database](../../../img/database-access/guides/redshift/cluster-user-db@2x.png)
+## Step 2/5. Create Teleport user
+
+(!docs/pages/includes/database-access/create-user.mdx!)
+
+## Step 3/5. Configure IAM
+
+### Create IAM policy for Teleport
+
+Teleport needs AWS IAM permissions to be able to:
+
+- Discover and register Redshift databases.
+- Manage IAM user or IAM role policies.
+
+Go to the [Policies](https://console.aws.amazon.com/iamv2/home#/policies) page
+and create a managed IAM policy for the database agent.
+
+<Tabs>
+  <TabItem label="Redshift / IAM user">
+  Use this policy if your Teleport database agent runs as IAM user (for example,
+  uses AWS credentials file).
+  ```json
+  {
+      "Version": "2012-10-17",
+      "Statement": [
+          {
+              "Effect": "Allow",
+              "Action": [
+                  "redshift:DescribeClusters",
+                  "iam:GetUserPolicy",
+                  "iam:PutUserPolicy",
+                  "iam:DeleteUserPolicy"
+              ],
+              "Resource": "*"
+          }
+      ]
+  }
+  ```
+  </TabItem>
+  <TabItem label="Redshift / IAM role">
+  Use this policy if your Teleport database agent runs as IAM role (for example,
+  on an EC2 instance with attached IAM role).
+  ```json
+  {
+      "Version": "2012-10-17",
+      "Statement": [
+          {
+              "Effect": "Allow",
+              "Action": [
+                  "redshift:DescribeClusters",
+                  "iam:GetRolePolicy",
+                  "iam:PutRolePolicy",
+                  "iam:DeleteRolePolicy"
+              ],
+              "Resource": "*"
+          }
+      ]
+  }
+  ```
+  </TabItem>
+</Tabs>
+
+### Create IAM permission boundary for Teleport
+Since Teleport will be managing its own IAM policies for access to Redshift
+databases, you need to create a permission boundary to limit its effective
+range of permissions.
+
+Create another managed policy that will serve as a permission boundary on the
+same [Policies](https://console.aws.amazon.com/iamv2/home#/policies) page.
+
+In addition to the set of permissions you created above, the boundary should
+also include `redshift:GetClusterCredentials` which will grant your Teleport
+agent the permssion to generate [temporary
+credentials](https://docs.aws.amazon.com/redshift/latest/mgmt/generating-user-credentials.html)
```

Comment:
Do we need the `[temporary credentials]` link here? I think we should prevent the reader from leaving the page as much as possible. We can add this link to the "Next steps" section.

Reply:
moved to "Next steps". though have to copy content from the included page otherwise the new list item has some bigger gaps from the included list.